### ISSUE TITLE
Error while trying to get nested data

### ISSUE THREAD

**Issue by alex-danilin** [r-dbi/bigrquery#168]
Hi.

I'm getting an error with this code while trying to get data with nested fields:

```
# Load libraries.
library(bigrquery)

# Set BigQuery project, query and get data from BQ.
bqProject <- "bq_project"

bqQuery <- "
SELECT
  *
FROM
`bq_project.OWOXBI_Streaming.streaming_20170630`
"
bqData <- query_exec(bqQuery, bqProject, max_pages = Inf, use_legacy_sql = FALSE)
```

The text of the error:
0 bytes processed
Error: Don't know how to convert type record
In addition: Warning message:
In matrix(data, nrow = length(types)) :
  data length [132985] is not a sub-multiple or multiple of the number of rows [36]

My goal is to pull all data from the table and it is not possible without using standard SQL dialect.

**Comment by byapparov** [comment]
And the source of the problem will be solved by this: #145

**Comment by hadley** [comment]
As @byapparov points out this is because bigrquery does not currently support nested fields.
